docs/testing/022-执行器安装按钮-测试.md

```
@@ -0,0 +1,111 @@
+const row = page.locator('[data-testid="executor-row-claudecode"]');
+
+// 安装按钮
+const installBtn = row.locator('[data-testid="executor-install-button"]');
+
+// Drawer
+const drawer = page.locator('.ant-drawer-content');
+```
+
+# 4. 测试运行方式
+
+### 4.1 单元测试
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)